example/spec/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -427,21 +419,28 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
     it('updates the UMM-G JSON file in S3 with new paths', async () => {
       const updatedUmm = await getUmmObject(newS3UMMJsonFileLocation);
 
-      const relatedUrlDifferences = updatedUmm.RelatedUrls.filter((urlObject) => {
-        // Skip non-science URLs and public S3 URLs
-        if (!isUMMGScienceUrl(urlObject.URL) ||
-            urlObject.URL.match(/s3\.amazonaws\.com/)) {
-          return false;
-        }
-        const relatedUrl = new URL(urlObject.URL);
-        relatedUrl.host = process.env.DISTRIBUTION_ENDPOINT;
-        return !originalUmmUrls.includes(relatedUrl.toString());
-      });
+      const expectedToChange = updatedUmm.RelatedUrls
```

Comment:
Rename to `expectedToChangeUrls`? Might be just me but I got easily confused reading this given the fact that `expect` is a reserved word for Jasmine

Reply:
Yes, because I filtered and mapped after I named...